spec/support/suspenders.rb

```diff
@@ -12,9 +12,10 @@ def create_tmp_directory
   def run_suspenders(arguments = nil)
     Dir.chdir(tmp_path) do
       Bundler.with_clean_env do
-        ENV['TESTING'] = '1'
-
-        %x(#{suspenders_bin} #{APP_NAME} #{arguments})
+        `
```

Comment:
Oh okay. I think that instead of having to set `PATH` inside of the command itself, you could set it outside. I think that if you set `ENV` outside of running a command, those environment variables will be set within the shell that the command runs inside of. I'm not 100% sure. But a cleaner way might be to simply call `add_fakes_to_path` before running the command.